packages/esm-patient-list-management-app/src/lists-dashboard/lists-dashboard.component.tsx

```diff
@@ -61,14 +59,11 @@ const ListsDashboard: React.FC = () => {
 
   const tableHeaders = [
     { id: 1, key: 'display', header: t('listName', 'List name') },
+    { id: 2, key: 'type', header: t('listType', 'List type') },
```

Comment:
Beyond this one [screen](https://zpl.io/VDNgdJv), I'm not sure that we should explicitly hide the list type column for any of the list tabs. Happy to change this back once get more clarity.